octobot_trading/personal_data/orders/order.py

```diff
@@ -136,6 +138,16 @@ def update(self, symbol, order_id="", status=enums.OrderStatus.OPEN,
             self.currency, self.market = self.exchange_manager.get_exchange_quote_and_base(symbol)
             self.symbol = symbol
 
+        if quantity_currency is None:
+            if self.quantity_currency is None and self.symbol is not None:
+                self.quantity_currency = order_util.get_order_quantity_currency(
+                    self.exchange_manager,
+                    self.symbol,
+                    self.side
```

Comment:
Shouldn't it be self.position_side ?

Reply:
we can use `order.get_position_side` here if we are in futures trading